napi.h

```diff
@@ -538,6 +538,9 @@ namespace Napi {
    /// Get a public Symbol (e.g. Symbol.iterator).
    static Symbol WellKnown(napi_env, const std::string& name);
 
+   // Create a symbol in the global registry;
+   static Symbol For(napi_env env, const std::string& name);
```

Comment:
Do we need some other overloads for this methods?
```cpp
static Symbol For(napi_env env,  const char* description = nullptr);
static Symbol For(napi_env env,  String description);
static Symbol For(napi_env env, napi_value description);
```
What do you think about?

Reply:
Hmm yeah I think that will be a good idea. Will add them in soon.